### ISSUE TITLE
Upgrading docs should reference `import` method.

### ISSUE THREAD

**Issue by jaridmargolin** [mozilla-neutrino/neutrino-dev#131]
The `import` method is actually a closer resemblance to the functionality people may desire from the old constructor method as it is capable of resolving middleware strings based on passed `options.root`.

```
Since the Neutrino API no longer accepts preset strings in its constructor, you must now pass presets and middleware as code/functions to the .use() method. This means the API no longer does preset module resolution. This has been moved to the CLI.
// v4 API
const Neutrino = require('neutrino');
const api = new Neutrino(['neutrino-preset-node', 'neutrino-preset-mocha']);

// v5 API
const Neutrino = require('neutrino');
const api = new Neutrino();

api.use(require('neutrino-preset-node'));
api.use(require('neutrino.preset-mocha'));
```

**Comment by eliperelman** [comment]
In my functional PR, I have renamed `import` to the similar functionality of `requiresAndUses`. This seems more explicit than `import`, without the potential for name collision in the future.

**Comment by eliperelman** [comment]
Resolved with #140.
